packages/paste-website/src/pages/getting-started/inclusive-design/index.mdx

```diff
@@ -0,0 +1,279 @@
+      </ListItem>
+      <ListItem>Provide alternatives and/or descriptions for complex visualizations</ListItem>
+      <ListItem>Include only clear, meaningful animations</ListItem>
+      <ListItem>Do not rely on color as the only means of differentiation of visual elements</ListItem>
+      <ListItem>Use the same design patterns to solve the same problems</ListItem>
+      <ListItem>Translate / spell out acronyms the first time you use them</ListItem>
+      <ListItem>Give all form elements permanently visible labels</ListItem>
+      <ListItem>Give grouped form elements group labels</ListItem>
+      <ListItem>Place labels above form elements</ListItem>
+      <ListItem>Employ well-balanced, highly legible fonts (not too complex or elaborate)</ListItem>
+      <ListItem>
+        Do not use very thin font faces{' '}
+        <Anchor href="http://www.telegraph.co.uk/science/2016/10/23/internet-is-becoming-unreadable-because-of-a-trend-towards-light/">
+          (Why it matters)
+        </Anchor>
+      </ListItem>
+      <ListItem>Make UI controls look like controls; give them strong perceived affordance</ListItem>
+      <ListItem>Underline links — at least in body copy</ListItem>
+      <ListItem>Avoid pure white or pure black shades</ListItem>
+      <ListItem>Mark invalid fields clearly and provide associated error messages</ListItem>
+      <ListItem>Indicate swipe gesture support clearly, and provide simple tap-based alternatives</ListItem>
+      <ListItem>
+        Avoid time constraints where possible; provide a clear warning and option to extend where not possible
+      </ListItem>
+      <ListItem>Label and describe the same things with the same terminology</ListItem>
+      <ListItem>Do not instate "infinite scroll" by default; provide buttons to load more items</ListItem>
+      <ListItem>
+        Avoid justified body text <Anchor href="https://www.w3.org/TR/WCAG20-TECHS/F88.html">(Why it matters)</Anchor>
+      </ListItem>
+      <ListItem>
+        Provide enough spacing between lines of text (`line-height`){' '}
+        <Anchor href="https://www.w3.org/TR/WCAG20-TECHS/C21.html">(Why it matters)</Anchor>
+      </ListItem>
+      <ListItem>
+        Avoid all-caps text{' '}
+        <Anchor href="https://github.com/humanmade/hm-pattern-library/issues/75">(Why it matters)</Anchor>
+      </ListItem>
+      <ListItem>
+        Ensure that content is written as clearly and simply as possible{' '}
+        <Anchor href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-supplements.html">(Why it matters)</Anchor>
+      </ListItem>
+      <ListItem>Provide descriptive captions for figures</ListItem>
+      <ListItem>Warn users of links that have unusual behaviors, like linking off-site, or loading a new tab</ListItem>
+      <ListItem>Use textual labels to make voice activation cues obvious</ListItem>
+      <ListItem>Do not mark up subheadings/straplines with separate heading elements</ListItem>
```

Comment:
This could be moved to or duplicated to the engineering section.

(i also had to google what straplines are)